modules/reference/pages/properties.adoc

```diff
@@ -0,0 +1,6361 @@
+= Redpanda Configuration Properties 
+:description: Redpanda configuration properties. 
+
+== Broker
+
+Broker configuration properties are applied individually to each broker in a cluster. 
+
+IMPORTANT: After you change a broker-level property setting, you must restart the broker for the change to take effect. 
+
+---
+
+=== admin
+
+Address and port of admin server
+
+*Requires Restart:* Yes
+
+*Nullable:* No
+
+*Visibility:* user
+
+*Type:* #/definitions/model::broker_endpoint
+
+*Default:* '{model::broker_endpoint(net::unresolved_address("127.0.0.1", 9644))}'
+
+=== admin_api_doc_dir
+
+Admin API doc directory
+
+*Requires Restart:* Yes
+
+*Nullable:* No
+
+*Visibility:* user
+
+*Type:* string
+
+*Default:* '/usr/share/redpanda/admin-api-doc'
+
+=== admin_api_tls
+
+TLS configuration for admin HTTP server
+
+*Requires Restart:* Yes
+
+*Nullable:* No
+
+*Visibility:* user
+
+*Type:* #/definitions/endpoint_tls_config
+
+*Default:* None
+
+=== cloud_storage_cache_directory
+
+Directory for archival cache. Should be present when `cloud_storage_enabled` is present
+
+*Requires Restart:* Yes
+
+*Nullable:* Yes
+
+*Visibility:* user
+
+*Type:* string
+
+*Default:* None
+
+=== crash_loop_limit
+
+Maximum consecutive crashes (unclean shutdowns) allowed after which operator intervention is needed to startup the broker. Limit is not enforced in developer mode.
+
+*Requires Restart:* Yes
+
+*Nullable:* Yes
+
+*Visibility:* user
+
+*Type:* integer
+
+*Accepted values:* [0, 4294967295]
+
+*Default:* 5
+
+=== data_directory
+
+Place where redpanda will keep the data
+
+*Requires Restart:* Yes
+
+*Nullable:* No
+
+*Visibility:* user
+
+*Type:* string
+
+*Default:* None
+
+=== developer_mode
+
+Skips most of the checks performed at startup, not recomended for production use
+
+*Requires Restart:* Yes
+
+*Nullable:* No
+
+*Visibility:* tunable
+
+*Type:* boolean
+
+*Default:* False
+
+=== emergency_disable_data_transforms
+
+Override the cluster enablement setting and disable WebAssembly powered data transforms. Only used as an emergency shutoff button.
```

Comment:
**ENG:** What, exactly, is the "cluster enablement setting"? Ww should call it out by name and link to it.